erpnext_com/www/docs/user/manual/en/loan-management/process-loan-interest-accrual.md

```diff
@@ -0,0 +1,35 @@
+<!-- add-breadcrumbs -->
+# Process Loan Interest Accrual
+> Introduced in Version 13
+
+**Process Loan Interest record is created on every loan Interest accrual cycle and is also used for manually processing loan interest accrual.**
+
+<img class="screenshot" alt="Loan Application" src="{{docs_base_url}}/assets/img/loan-management/process-loan-interest-accrual-flow.png">
+
+To access the Process Loan Interest Accrual list, go to:
+> Home > Loan Management > Loan Processes > Process Loan Interest Accrual
+
+Loan Interest will be accrued monthly on first day of every month for demand loans and one day before payment day for term loans by a background job but if you want to accrue interest manually against a loan then it can be done using Process Loan Interest Accrual
```

Comment:
```suggestion
Loan Interest will be accrued monthly on the first day of every month for demand loans and one day before payment day for term loans by a background job. If you want to accrue interest manually against a loan then it can be done using Process Loan Interest Accrual.
```